src/core/client/stream/test/comments/stream/postReply.spec.tsx

```diff
@@ -326,36 +326,3 @@ it("handle disabled commenting error", async () => {
   expect(rte.props.disabled).toBe(true);
   expect(within(form).getByText("Submit").props.disabled).toBe(true);
 });
-
-it("handle story closed error", async () => {
```

Comment:
I deleted this test because we can no longer see the stream when it is closed.

Reply:
If the story is closed while the user has the comment stream opened, it will occur.

Reply:
Looking into this, think you're right. Not sure why I was getting a node not mounted search error with this.

Reply:
Is fixed.